cmd/client/commandv2/general.go

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commandv2
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/megaease/easegress/cmd/client/general"
+	"github.com/megaease/easegress/pkg/api"
+	"github.com/megaease/easegress/pkg/util/codectool"
+	"github.com/spf13/cobra"
+)
+
+// CompletionCmd returns completion command to generate completion script for the specified shell (bash or zsh).
+func CompletionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "completion [bash|zsh]",
+		Short:                 "Generates completion script for the specified shell (bash or zsh)",
+		DisableFlagsInUseLine: true,
+		ValidArgs:             []string{"bash", "zsh"},
+		Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+		Run: func(cmd *cobra.Command, args []string) {
+			switch args[0] {
+			case "bash":
+				cmd.Root().GenBashCompletion(os.Stdout)
+			case "zsh":
+				cmd.Root().GenZshCompletion(os.Stdout)
+			}
+		},
+	}
+	return cmd
+}
+
+// HealthCmd returns health command.
+func HealthCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "health",
+		Short:   "Probe Easegress health",
+		Example: createExample("Probe Easegress health", "egctl health"),
+		Run: func(cmd *cobra.Command, args []string) {
+			_, err := handleReq(http.MethodGet, makeURL(general.HealthURL), nil, cmd)
+			if err != nil {
+				general.ExitWithError(err)
+			}
+			if !general.CmdGlobalFlags.DefaultFormat() {
+				general.PrintBody([]byte("{msg: OK}"))
```

Comment:
`{"message": "OK"}` should be a valid json. `{msg: OK}` is invalid.

Reply:
```
➜  egctl health                
OK
➜  egctl health -o yaml 
msg: OK
➜  egctl health -o json 
{msg: OK}%           
```
It's actually valid... because json will see non-number, non-bool value as string. But i will update it to make it more clearly...